python-checks/src/main/java/org/sonar/python/checks/cdk/CdkUtils.java

```diff
@@ -182,6 +193,64 @@ public static Optional<ResolvedKeyValuePair> getKeyValuePair(SubscriptionContext
     return element.is(Tree.Kind.KEY_VALUE_PAIR) ? Optional.of(ResolvedKeyValuePair.build(ctx, (KeyValuePair) element)) : Optional.empty();
   }
 
+
+  //--------------------------------------------------
+  //-------------- AWS IAM related utils -------------
+  //--------------------------------------------------
```

Comment:
Since we already discussed moving many of the helpers to a general PythonUtils class at the end of the sprint, I wanted to open a section that is exclusive to CdkIam. I didn't want to create an extra class, but maybe this makes sense.